arex-instrumentation-api/src/main/java/io/arex/inst/runtime/service/DataService.java

```diff
@@ -29,17 +30,23 @@ public String query(String data, MockStrategyEnum mockStrategy) {
         return saver.query(data, mockStrategy);
     }
 
-    public static class Builder {
-
-        private DataCollector collector;
+    public String queryAll(String data) {
+        return saver.queryAll(data);
+    }
 
-        public Builder setDataCollector(DataCollector collector) {
-            this.collector = collector;
-            return this;
+    public static void setDataCollector(List<DataCollector> collectors) {
+        if (CollectionUtil.isEmpty(collectors)) {
+            return;
         }
-
-        public void build() {
-            INSTANCE = new DataService(this.collector);
+        DataCollector collector = collectors.get(0);
+        if (Config.get().isLocalStorage()) {
+            for (DataCollector dataCollector : collectors) {
+                if (ArexConstants.STANDALONE_MODE.equals(dataCollector.mode())) {
```

Comment:
Suggest add a default method for the interface, then collectors are sorted in reverse order by `supportLocal`, return the first one.

```
default boolean supportLocal() { return false; }
```